ppa/archive/templates/archive/snippets/search_result.html

```diff
@@ -14,10 +14,17 @@
                     {% if page_obj %}
                         <p class="result-number">{{ page_obj.start_index|add:forloop.counter0 }}</p>
                     {% endif %}
+                    <div class="work-type" aria-label="{{ item.work_type }}"></div>
```

Comment:
does this make sense for full works? or could we just omit it for for full work?

Reply:
I tried testing the search results page with voice over because I wanted to hear what it sounded like, and I couldn't tab through the search form into the results — it goes from the sort by option back to the browser back button.  Am I doing something wrong?